packages/explorer/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@latticexyz/explorer",
+  "version": "2.1.0",
+  "private": true,
+  "description": "Explorer is a tool for visually exploring and manipulating the state of worlds",
+  "bin": {
+    "explorer": ".next/standalone/packages/explorer/server.js"
+  },
+  "scripts": {
+    "build": "next build && cp -r .next/static .next/standalone/packages/explorer/.next && pnpm link .",
+    "dev": "next dev",
+    "lint": "next lint",
+    "start": "node .next/standalone/packages/explorer/server.js"
+  },
+  "dependencies": {
+    "@hookform/resolvers": "^3.9.0",
+    "@latticexyz/common": "workspace:*",
+    "@latticexyz/protocol-parser": "workspace:*",
+    "@latticexyz/schema-type": "workspace:*",
+    "@latticexyz/store": "workspace:*",
+    "@latticexyz/world": "workspace:*",
+    "@radix-ui/react-checkbox": "^1.1.1",
+    "@radix-ui/react-dropdown-menu": "^2.1.1",
+    "@radix-ui/react-label": "^2.1.0",
+    "@radix-ui/react-select": "^2.1.1",
+    "@radix-ui/react-separator": "^1.1.0",
+    "@radix-ui/react-slot": "^1.1.0",
+    "@radix-ui/react-toast": "^1.2.1",
+    "@radix-ui/themes": "^3.0.5",
+    "@tanstack/react-query": "^5.51.3",
+    "@tanstack/react-table": "^8.19.3",
+    "@uidotdev/usehooks": "^2.4.1",
+    "@wagmi/core": "^2.12.1",
+    "better-sqlite3": "^8.6.0",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.1.1",
+    "lodash": "^4.17.21",
```

Comment:
I have PTSD from lodash and bundle sizes. Looks like we only use `camelCase`, so wondering if there's a lightweight version of this that we can include as a function and use throughout?

Reply:
Ah good point, not worth importing just for one function, left it during prototyping. I'll implement our own function for this.